src/utilities.js

```diff
@@ -68,6 +76,10 @@ export function hasOwnProperty (obj, key) {
   return obj && Object.prototype.hasOwnProperty.call(obj, key)
 }
 
+export function isNumber2 (value) {
```

Comment:
unlike the object copying, the updated `isNumber` and `isInteger` implementation continue to be faster